layouts/docs/baseof.html

```diff
@@ -25,7 +25,7 @@
             {{ if (and (not .Params.hide_feedback) (.Site.Params.ui.feedback.enable) (.Site.GoogleAnalytics)) }}
               {{ partial "feedback.html" .Site.Params.ui.feedback }}
             {{ end }}
-            <div class="text-muted mt-5 pt-3 border-top">{{ partial "page-meta-lastmod.html" . }}</div>
+            <div class="text-muted">{{ partial "page-meta-lastmod.html" . }}</div>
```

Comment:
Hi @tengqm .
What do you think about removing the first `<div></div>`? The partial `page-meta-lastmod.html` from the theme includes this div element?

Reply:
checking

Reply:
@kbhawkey you are right. The partial already contains a DIV. An extra DIV is not needed. Fixed.